src/main/java/pl/isa/biblioteka/menu/BooksBorrowMenu.java

```diff
@@ -0,0 +1,106 @@
+package pl.isa.biblioteka.menu;
+
+import pl.isa.biblioteka.user.LogUser;
+import pl.isa.biblioteka.user.Person;
+
+import java.util.Scanner;
+
+import static pl.isa.biblioteka.books.BorrowBooks.*;
+
+public class BooksBorrowMenu {
+    public static Scanner scanner = new Scanner(System.in);
+
+    public void mainLoop() {
+        System.out.println("Co chcesz zrobić");
+        Option option = null;
+        do {
+            printMenu();
+            try {
+                option = chooseOption();
+                executeOption(option);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        } while (option != Option.EXIT);
+    }
+
+    private Option chooseOption() {
+        int option = scanner.nextInt();
+        scanner.nextLine();
+        return Option.fromInt(option);
+    }
+
+    private void executeOption(Option option) {
+        LogUser logUser = new LogUser();
+        switch (option) {
+            case BORROW_BOOK -> {
+                if ((addBookToPerson())) {
+                    System.out.println("Książka została wypożyczona");
+                } else {
+                    System.out.println("Nie posiadamy książki o podanym tytule");
+                }
+            }
+            case RETURN_BOOK -> {
+                if (returnBook()) {
+                    System.out.println("Książka została zwrócona");
+                } else {
+                    System.out.println("Użytkoniku nie posiadasz takiej kiążki");
+                }
+                ;
+            }
+            case SHOW_AVAILABLE_BOOK -> booksEdit.showAllAvailableBooks();
+            case SHOW_BORROWED_BOOK -> {
+                if (!logUser.getPersonBooks().isEmpty()) {
+                    logUser.getPersonBooks().forEach(
+                            (book) -> System.out.println("Tytuł: " + book.getTitle() + " Autor: " + book.getAuthor())
+                    );
+                } else {
+                    System.out.println("Nie wypożyczono jeszcze żadnej książki");
+                }
+            }
+            case SHOW_SORTED_BOOK -> sortByCategory();
+            case EXIT -> close();
+        }
+    }
+
+    private void close() {
+        System.out.println("Bye Bye!");
```

Comment:
Nie widzę co close() robi, żeby zamknąć program, poza wypisaniem tekstu do konsoli ;)

Reply:
Usunięto tekst próbny